sound.js

```diff
@@ -13,6 +13,29 @@ function isRelativePath(path) {
   return !/^(\/|http(s?)|asset)/.test(path);
 }
 
+const registerOnPlay = function(onPlayListener) {
```

Comment:
FYI, two issues:

1) If it sits here, this method will be shared across all the `Sound` instances.
2) You need to come up with how to deal with EventEmitter working with multiple `Sound` instances. We shouldn't assume that there is only a single instance of `Sound` instance. Even for our app, we have several player instances at a time.

@ghsdh3409 Please take a look at this issue in case you further continue to use this event emitter in JS level.

Reply:
for that issue, when initializing new `Sound` instance, I added `this.registerOnPlay = registerOnPlay.bind(this);` to make `registerOnPlay` specific to each instances. Wouldn't this work?

Reply:
@sunjae1294 But, in the native Android code, the event is sent globally. Thus, if there are several `Sound` instance, the event can be sent to several instances.

Reply:
@ghsdh3409 Thanks!!! never thought of that!! I think I can solve it by using `key` variable passed between js and native code.